src/Components/Alert/Components/Filters/QuickMultipleSelectAlertFilter.tsx

```diff
@@ -51,22 +49,20 @@ export const QuickMultipleSelectAlertFilter: FC<QuickMultipleSelectAlertFilterPr
       <Spacer y={2} />
 
       <GridColumns>
-        <ShowMore expanded={expanded}>
```

Comment:
@olerichter00 I've removed the "ShowMore" as we're not using it yet and it was causing the spacing on the Medium filter when expanded.

Reply:
Good idea to remove it 👍 In case we need it at some point, we can bring it back.